### PR TITLE
Add auto comment to pull requests

### PR DESCRIPTION
**Pull request by mathiasandresen**
Added a Github Action that adds the review checklist to the comment.
This was a recommendation, which was positively voted for in the retrospective.

### CODE REVIEW

**Comment by mathiasandresen** [comment]
This is not changes to the code base, but is only github related

**Comment by mathiasandresen** [comment]
I know that the Azure pipeline is failing, but this is not related to this, and is fixed in another PR

File: .github/workflows/comment.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  comment_checklist:
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: comment PR
+      uses: unsplash/comment-on-pr@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        msg: |
+          Please use the following checklist. Feel free to check it off as you go along.
```
Comment (T-Hauge) [inline .github/workflows/comment.yml]:
```suggestion
          Please use the following checklist.
```

Comment (T-Hauge) [reply .github/workflows/comment.yml]: Since the checklist is shared for all, it might lead to confusing to state this.

Comment (T-Hauge) [reply .github/workflows/comment.yml]: Maybe even remove the checkboxes completely.

**Comment by pegede17** [comment]
Muligvis ligegyldigt og kan godt være fordi jeg ikke forstår det helt, men i kommentaren i linje 6 siger at det er master branchen det vedrører, men i linje 9 står der develop. Er dette meningen?

Derudover virker nogle af punkterne lidt mærkelige at chekke af.
Fx. Kunne man godt tænke at man kun skal chekke "Does it look like the code has bugs?" af, hvis der er bugs, og det er vel meningen at den skal chekkes af når der IKKE er bugs.

**Comment by mathiasandresen** [comment]
unintentional behaviour

**Pull request closed without merging**